--- link-grammar-5.12.3/bindings/perl/Makefile.am.orig	2023-06-02 17:24:16.764904515 +0200
+++ link-grammar-5.12.3/bindings/perl/Makefile.am	2023-06-02 17:24:58.628011057 +0200
@@ -52,7 +52,7 @@ if OS_NETBSD
 AVOID_VERSION = -avoid-version
 endif
 
-clinkgrammar_la_LDFLAGS = -version-info $(AVOID_VERSION) @VERSION_INFO@ \
+clinkgrammar_la_LDFLAGS = -avoid-version \
 								  $(PERL_LDFLAGS) -module -no-undefined
 clinkgrammar_la_LIBADD = $(top_builddir)/link-grammar/liblink-grammar.la
 
--- link-grammar-5.12.3/bindings/python/Makefile.am.orig	2023-06-02 06:44:56.982717281 +0200
+++ link-grammar-5.12.3/bindings/python/Makefile.am	2023-06-02 17:23:33.231807020 +0200
@@ -88,7 +88,7 @@ if OS_NETBSD
 AVOID_VERSION = -avoid-version
 endif
 _clinkgrammar_la_LDFLAGS =                        \
-    -version-info @VERSION_INFO@ $(AVOID_VERSION) \
+    -avoid-version \
     $(PYTHON_LDFLAGS) -module -no-undefined $(PYMODULE_EXT)
 _clinkgrammar_la_LIBADD =                         \
     $(top_builddir)/link-grammar/liblink-grammar.la $(PYTHON_LIBS)
